density/tests/test_calculate_capacity.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+
+from density import calculate_capacity
+
+
+class TestCalculateCapacity(unittest.TestCase):
+
+    # testing logic in calulate_capacity()
+    def test_calculate_capacity(self):
+        # test version of cap_data
+        cap_test = [{'capacity': 100, 'group_name': 'Butler Library 6'},
```

Comment:
@natebrennand I'm getting pep8 compliance issues about the lines here - if I don't like up the entries (indent more), I get an indentation error, but when I indent like they ask for pep8 complains about the lines being too long. Is this a problem/ should I worry about it?
![screen shot 2015-04-03 at 4 29 09 pm](https://cloud.githubusercontent.com/assets/8763732/6988363/a2190c9a-da1e-11e4-9117-61b5859ae6a8.png)